_data/simple-icons.json

```diff
@@ -9719,7 +9724,8 @@
         {
             "title": "Veritas",
             "hex": "B1181E",
-            "source": "https://my.veritas.com/cs/groups/partner/documents/styleguide/mdaw/mdq5/~edisp/tus3cpeapp3855186572.pdf"
+            "source": "https://www.veritas.com/",
```

Comment:
There are no vector versions of their logo there.
I found one in the Corporate Overview linked from https://www.veritas.com/company/corporate-profile

Reply:
Yes, you can open the SVG using Inkscape. There is Vector version of their logo in it.